Cargo.toml

```diff
@@ -15,6 +15,10 @@ proc-macro = true
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[features]
+ra-friendly = []
+check-friendly = []
```

Comment:
I think these features deserve to be enabled by default, could you add that please?

Reply:
Yes, I originally didn't want to denature your “Language server-unfriendly: rust-analyzer is now also taking a nap.” :wink:
They are now enabled by default!

Reply:
Sorry for the last force-push, I rebased my commits on top of the updated main branch to fix the git conflicts.